pkg/kn/commands/service/configuration_edit_flags.go

```diff
@@ -434,6 +438,17 @@ func (p *ConfigurationEditFlags) Apply(
 		}
 	}
 
+	if cmd.Flags().Changed("scale") {
```

Comment:
I can comment lines 443-445 and/or lines 447-449 and all tests still pass.

This tells me you don’t have tests for `servinglib.UpdateMaxScale(...)` and `servinglib.UpdateMinScale(...)` failings. Might be easy to cover with a test with negative or 0 scale (perhaps).